decidim-debates/spec/system/debates_breadcrumbs_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "debates breadcrumbs" do
+  include_context "with a component"
+  let(:manifest_name) { "debates" }
+
+  let(:description) { generate_localized_description(:debate_description) }
+  let(:information_updates) { generate_localized_description(:information_updates) }
+  let(:instructions) { generate_localized_description(:instructions) }
+  let!(:debate) { create(:debate, component:, description:, information_updates:, instructions:) }
```

Comment:
Why adding these variables? Are they being used for something? 

```suggestion
  let!(:debate) { create(:debate, component:) }
```

Reply:
cleaned.